src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -559,24 +563,30 @@ const CreateImageWizard = () => {
 
   const { isBeta, isProd } = useGetEnvironment();
 
+  // Only allow oscap to be used in Beta even if the flag says the feature is
+  // activated.
+  const oscapFeatureFlag =
+    useFlag('image-builder.wizard.oscap.enabled') && isBeta;
```

Comment:
The feature flag for stage works great! But there will be a small fix needed if we want to make the step available only in beta.

```suggestion
    useFlag('image-builder.wizard.oscap.enabled') && isBeta();
```

Reply:
:smile: Good catch!!

Reply:
Fixed!